README.md

```diff
@@ -522,37 +563,60 @@
     if (test)
       return false;
 
+    // acceptable (only for very simple statements and early function return)
+    if (test) return false;
+
     // good
     if (test) {
       return false;
     }
 
-    // good
-    function () { return false; }
+    // bad
```

Comment:
Why did this get changed to bad? Curly braces are optional at worst, and never to be considered bad.